rcp/org.testeditor.rcp4.tcltestrun/src/org/testeditor/rcp4/tcltestrun/MavenExecutor.java

```diff
@@ -49,7 +51,16 @@
 	public int execute(String parameters, String pathToPom) {
 		System.setProperty("maven.multiModuleProjectDirectory", pathToPom);
 		MavenCli cli = new MavenCli();
-		int result = cli.doMain(parameters.split(" "), pathToPom, System.out, System.err);
+		List<String> params = new ArrayList<String>();
+		params.addAll(Arrays.asList(parameters.split(" ")));
+		String mavenSettings = System.getProperty("TE.MAVENSETTINGSPATH");
```

Comment:
We got the retrieval (and the constant) twice here. Maybe we can extract this code into a helper class?